src/main/resources/marc4bibframeAutority.yml

```diff
@@ -0,0 +1,65 @@
+authorityFieldRules:
+  '100':
+    - types: CONCEPT, PERSON
```

Comment:
One thing to keep in mind so that it won't be complex to implement  [MODLD-288](https://folio-org.atlassian.net/browse/MODLD-288)

When $v, $x, $y or $z is present in marc 100, then the types if the root resource will be  `CONCEPT`, `PERSON`. And then there will be another `FOCUS` edge pointing to the `Person` resource.


If $v, $x, $y or $z is not present in marc 100, then the type of the root resource will be just `PERSON`. There will not be any outgoing `FOCUS` edge to `Person` (as the root resource itself is a Person). See [MODLD-288](https://folio-org.atlassian.net/browse/MODLD-288) for details.